src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,14 +118,19 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.ofHours(TIME_SHIFT));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
+        try {
+            return Optional.of(LocalDate.parse(date, FORMATTER1));
```

Comment:
probably we have a ready-made formatter for this case?

Reply:
done